zkevm-circuits/src/root_circuit.rs

```diff
@@ -70,15 +139,22 @@ where
 {
     /// Create a `RootCircuit` with accumulator computed given a `SuperCircuit`
     /// proof and its instance. Returns `None` if given proof is invalid.
+    /// TODO support multiple snark proof aggregation
     pub fn new(
         params: &ParamsKZG<M>,
         super_circuit_protocol: &'a PlonkProtocol<M::G1Affine>,
```

Comment:
In the actual aggregation situation where we're dealing with multiple chunks, should `protocol`, `instances`, and `proof` all be Vec? So that the `flatten_super_circuit_instances` will actually contain concatenated instances from different chunks.